recipe/build.sh

```diff
@@ -15,12 +15,7 @@ export CXXFLAGS="${CXXFLAGS//-std=c++17/-std=c++11}"
 # value at run-time to get the expected amount of parallelism.
 export OPENBLAS_NUM_THREADS=1
```

Comment:
@tkralphs Should that part also be removed? Not sure if that's pulled in somewhere.

Reply:
I believe this is needed. Even though Blas is not a direct dependency, this just ensures that if it _is_ linked, it doesn't do anything in parallel. By default, OpenBlas does parallel I believe.